### PR TITLE
OPENEUROPA-342: Integrate pagination component

### PR DESCRIPTION
**Pull request by netlooker**

### CODE REVIEW

File: tests/Kernel/PagerTest.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    // Assert that the pager contains only the 'previous' page link.
+    $this->assertContains(self::PREVIOUS_PAGE_LINK_TEXT, $third_pager->text());
+    $this->assertNotContains(self::NEXT_PAGE_LINK_TEXT, $third_pager->text());
+    // Assert that the pager contains only one ellipsis element.
+    $ellipsis_count = $third_pager->filter('li.ecl-pager__item--ellipsis')->count();
+    $this->assertEquals(1, $ellipsis_count);
+    // Assert that the pager is set on the last page.
+    $first_page_number = $third_pager->filter('li.ecl-pager__item--current')->text();
+    $this->assertContains('Page 15', $first_page_number);
+
```
Comment (brummbar) [inline tests/Kernel/PagerTest.php]: No empty line here.

File: tests/Kernel/PagerTest.php [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    // Check the second variant (pager set on the first page).
+    $second_pager = $crawler->filter('nav:nth-of-type(2)');
+    // Assert that the pager contains only the 'next' page link.
+    $this->assertContains(self::NEXT_PAGE_LINK_TEXT, $second_pager->text());
+    $this->assertNotContains(self::PREVIOUS_PAGE_LINK_TEXT, $second_pager->text());
+    // Assert that the pager contains only one ellipsis element.
+    $ellipsis_count = $second_pager->filter('li.ecl-pager__item--ellipsis')->count();
+    $this->assertEquals(1, $ellipsis_count);
+    // Assert that the pager is set on the first page.
+    $first_page_number = $second_pager->filter('li.ecl-pager__item--current')->text();
```
Comment (brummbar) [inline tests/Kernel/PagerTest.php]: We are missing the testing of the links themselves. At the moment there is no check that the page array is present.

**Comment by netlooker** [comment]
I forgot to regenerate the patch using `npx patch-package @ec-europa/ecl-pagers` command.

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
         "build": "npm-run-all --parallel build:* -ln",
         "watch:scripts": "chokidar \"./sass/**/*.js\" -i \"/node_modules/\" -c \"eslint {path} & npm run build:scripts\"",
         "watch:styles": "chokidar \"./sass/**/*.scss\" -i \"/node_modules/\" -c \"stylelint {path} & npm run build:styles\"",
-        "watch": "npm-run-all --parallel watch:* -ln"
+        "watch": "npm-run-all --parallel watch:* -ln",
+        "prepare": "patch-package"
```
[the remark this answers is not in the document]
Comment (ademarco) [reply package.json]: This has been fixed.

**Pull request merged**